# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/nsIFrame.h
# Commit: 6141de941e71
# Full Hash: 6141de941e717cdb5485e1aa1ca00ffcd74f8a81
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2021-01-30 19:41:15
# Regressor Bug: 1683748
# File Overlap Count: 3
# Description:
#   Bug 1683748 - Support Grid/Flex/Table/Column layout for the rendered legend of a fieldset.  r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D101555
# ==============================================================================

diff -r 3cf752580f1b -r 6141de941e71 layout/generic/nsIFrame.h
--- a/layout/generic/nsIFrame.h	Fri Jan 29 19:20:24 2021 +0000
+++ b/layout/generic/nsIFrame.h	Sat Jan 30 13:47:10 2021 +0000
@@ -2378,6 +2378,11 @@
   bool IsPrimaryFrameOfRootOrBodyElement() const;
 
   /**
+   * @return true if this frame is used as a fieldset's rendered legend.
+   */
+  bool IsRenderedLegend() const;
+
+  /**
    * This call is invoked on the primary frame for a character data content
    * node, when it is changed in the content tree.
    */